ibis/backends/base/__init__.py

```diff
@@ -215,7 +215,7 @@ def _ipython_key_completions_(self) -> list[str]:
 
 
 # should have a better name
-class ResultHandler:
+class FileIOHandler:
```

Comment:
Isn't this class name part of the public API (for backend developers)? Doesn't this renaming a deprecation cycle with a temporary backward compat alias?

Reply:
I don't think so, this was introduced in `4.0` and it is extremely unlikely that someone has built a backend in the last month that is using this class.

@gforsyth Can you make this private with a leading underscore?

Reply:
yeah, will do